master.cfg

```diff
@@ -186,7 +186,8 @@ for worker in conf.workers:
             image=util.Property('docker_image'),
             autopull=True,
             alwaysPull=True,
-            hostconfig=dict(network_mode='host')
+            hostconfig=worker.hostconfig,
+            max_builds=worker.get('max_builds', 1)
```

Comment:
We should not limit `max_builds` by default.

Reply:
I intend to refactor workers in prod.toml to have 1 worker per each set of 4-5 cpus.